### PR TITLE
Fixed error with php 5.3.3 in travis

### PR DESCRIPTION
**Pull request by AntikCz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +27,6 @@ matrix:
       env: PHP_BIN=php-cgi
 
 install:
-  - composer self-update
```
Comment (stekycz) [inline .travis.yml]: This updated cause the error? It think it should remain in the build process.

Comment (AntikCz) [inline .travis.yml]: [Yes](https://github.com/composer/composer/blob/master/src/Composer/Command/SelfUpdateCommand.php#L136-L139). If you want use this command, you must add into composer.json [disable-tls](https://github.com/composer/composer/blob/master/src/Composer/Command/SelfUpdateCommand.php#L133)

**Comment by AntikCz** [comment]
Is second solution better?

**Comment by stekycz** [comment]
Great! Thank you a lot

**Pull request merged**